employees/common/strings.py

```diff
@@ -94,11 +94,12 @@ class ProjectReportDetailStrings(NotCallableMixin, Enum):
 
 
 class ReportValidationStrings(NotCallableMixin, Enum):
+    WORK_HOURS_MINUTES_ARE_INDIVISIBLE_BY_FIFTEEN = _("Possible values for minutes are 0, 15, 30 and 45")
     WORK_HOURS_SUM_FOR_GIVEN_DATE_FOR_SINGLE_AUTHOR_EXCEEDED = _(
         "Sum of work hours of all reports from a given day for single author must not exceed 24."
     )
     WORK_HOURS_MIN_VALUE_NOT_EXCEEDED = _(
-        "Minimum value for work hours for single report must not be less than 1 minute"
+        "Minimum value for work hours for single report must not be less than 15 minutes"
```

Comment:
Why not using `MIN_WORK_HOURS` constant here? Like this `f"must not be less than{ReportModelConstants.MIN_WORK_HOURS.total_seconds() / 60}"`.

Reply:
Done